### PR TITLE
Add jester suits

### PR DESCRIPTION
**Pull request by Morb0**
## About the PR 
New suits in AutoDrobe.

**Screenshots**

image image

**Changelog**

:cl:
- add: Added two new jester suits in AutoDrobe

### CODE REVIEW

**Comment by Elijahrane** [comment]
Are their inhands supposed to be clown suits?

**Comment by ZeroDayDaemon** [comment]
> Are their inhands supposed to be clown suits?

Most likely, assuming these are pulled from /tg/. I've noticed alot of tg stuff has generic offhands.

File: Resources/Prototypes/Entities/Clothing/Shoes/specific.yml [hunk 1]
```diff
@@ -130,3 +130,14 @@
   description: Stylish black shoes.
   components:
   - type: NoSlip
+
+- type: entity
+  parent: ClothingShoesBaseButcherable
+  id: ClothingShoesJester
+  name: jester shoes
+  description: A court jester's shoes, updated with modern squeaking technology.
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Clothing/Shoes/specific.yml:138]: these dont inherit from the clown shoes, so no squeak

File: Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml [hunk 2]
```diff
@@ -151,6 +151,28 @@
...
+  - type: Sprite
+    sprite: Clothing/Uniforms/Jumpsuit/jester.rsi
+  - type: Clothing
+    sprite: Clothing/Uniforms/Jumpsuit/jester.rsi
+
+- type: entity
+  parent: ClothingUniformBase
+  id: ClothingUniformJumpsuitJesterAlt
+  name: jester suit
+  description: A jolly dress, well suited to entertain your master, nuncle.
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml:169]: this should just inherit from the other jester suit

File: Resources/Prototypes/Entities/Clothing/Head/hats.yml [hunk 3]
```diff
@@ -478,3 +478,26 @@
...
+    - type: Sprite
+      sprite: Clothing/Head/Hats/jester.rsi
+    - type: Clothing
+      sprite: Clothing/Head/Hats/jester.rsi
+
+- type: entity
+  parent: ClothingHeadBase
+  id: ClothingHeadHatJesterAlt
+  name: jester hat
+  description: A hat with bells, to add some merriness to the suit.
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Clothing/Head/hats.yml:497]: same here

**Review by mirrorcult — DISMISSED** [review]

**Pull request merged**